packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @flow
```

Comment:
We don't want Flow annotations in Typescript files

Reply:
Removed in newest commit